src/output_counties.R

```diff
@@ -219,7 +218,7 @@ mseFunc <- function(trueVal,keyword){
 }
 
 
-nVec <- c(100,500,1000, 10000)
+nVec <- c(100, 500, 1000, 10000)
```

Comment:
@rosenbat2 this is also hardcoded, but it includes 10k so it should be correct (i.e. consistent with the simulation sample sizes in `fileReadVec <-c('2019-10-21 18:43:53')`).